css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,45 @@
+body {
+  font-family: "DM Sans", sans-serif;
+}
+
+div.row > h1 {
+  text-align: center;
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  margin: 0 90px;
+}
+
+.col {
+  flex-basis: calc(100%/5);
+}
+
+img {
+  height: 330px;
+  width: 210px;
+}
```

Comment:
You should avoid setting both a fixed width & height on an element, because it screws up the aspect ratio, and causes the images to stretch. If you're going to use a set px value, you should only set it for one. Something like 
```
img {
  width: 100%;
  height: 330px;
}
```